CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.12.0 (1 December, 2022)
+
+### Enhancements
+ * Provide HTML support for Qute HTML templates. See [#483](https://github.com/redhat-developer/vscode-quarkus/issues/483).
+ * Add new snippet for `@QuarkusIntegrationTest`. See [quarkus-ls#754](https://github.com/redhat-developer/quarkus-ls/issues/754).
+ * Generic support for Java data model in Qute templates. See [quarkus-ls#503](https://github.com/redhat-developer/quarkus-ls/issues/503).
+ * Syntax coloration for operator of `#case`. See [#537](https://github.com/redhat-developer/vscode-quarkus/issues/537).
+ * Show documentation on hover of operator for `#case`, `#is`. See [quarkus-ls#716](https://github.com/redhat-developer/quarkus-ls/issues/716).
+ * Clickable inlay hint for Java type in Qute templates. See [#533](https://github.com/redhat-developer/vscode-quarkus/pull/533).
+ * Validation, completion & definition support for enum in `#switch` section. See [quarkus-ls#689](https://github.com/redhat-developer/quarkus-ls/issues/689), [#690](https://github.com/redhat-developer/quarkus-ls/issues/690), [quarkus-ls#691](https://github.com/redhat-developer/quarkus-ls/issues/691).
+  * Display property expression evaluation as inlay hint. See [vscode-microprofile#108](https://github.com/redhat-developer/vscode-microprofile/pull/108).
+ * Added Qute code actions for similar text suggestions. See [quarkus-ls#602](https://github.com/redhat-developer/quarkus-ls/issues/602).
+ * Code actions to create Java field / getter method / template extension. See [quarkus-ls#536](https://github.com/redhat-developer/quarkus-ls/issues/536), [quarkus-ls#676](https://github.com/redhat-developer/quarkus-ls/issues/676), [quarkus-ls#677](https://github.com/redhat-developer/quarkus-ls/issues/677).
+ * Display Javadoc on hover for property/method . See [#452](https://github.com/redhat-developer/quarkus-ls/issues/452).
```

Comment:
`quarkus-ls#425`

Reply:
Done. Got a bit lost when looking for 425 instead of 452 :P